container/ubuntu16_04/bazel/container.yaml

```diff
@@ -22,13 +23,22 @@ commandTests:
 - name: 'check-openssl'
   command: ['bash', '-c', 'openssl version']
   expectedOutput: ['OpenSSL .*']
-- name: 'bazel'
-  command: ['bazel']
-  expectedOutput: ['Usage: bazel <command> <options>.*']
+- name: 'check-git'
+  command: ['bash', '-c', 'git --version']
+  expectedOutput: ['git version .*']
+- name: 'check-unzip'
+  command: ['bash', '-c', 'unzip']
+  expectedOutput: ['.*Usage: unzip .*']
 
 # File existence tests
 fileExistenceTests:
 - name: 'OpenJDK'
   isDirectory: true
   path: '/usr/lib/jvm/java-8-openjdk-amd64'
   shouldExist: true
+
+# TODO(xingao) Test entry point once rules_docker is updated with latest
```

Comment:
this check should probably be in bazel.yaml

Reply:
I feel we set this entrypoint to the container, not bazel layer. Maye it's more appropriate to set it here.